main/inc/lib/extra_field.lib.php

```diff
@@ -793,67 +741,15 @@ public function set_extra_fields_in_form(
                             $field_details['field_display_text'] = get_lang($field_details['field_display_text']);
                         }
 
-                        // chzn-select doesn't work for sessions??
-                        $form->addElement(
-                            'select',
-                            'extra_'.$field_details['field_variable'],
-                            $field_details['field_display_text'],
-                            $options,
-                            array('id' => 'extra_'.$field_details['field_variable'])
+                        //chzn-select doesn't work for sessions??
+                        $form->addElement('select', 'extra_' . $field_details['field_variable'],
+                            $field_details['field_display_text'], $options,
+                            array('class' => 'chzn-select', 'id' => 'extra_' . $field_details['field_variable'])
                         );
 
-                        if ($optionsExists && $field_details['field_loggeable'] && !empty($defaultValueId)) {
```

Comment:
When I use the removed lines, I get this error : `Fatal error: Call to a member function getRepository() on a non-object in /var/www/chamilo19x/main/inc/lib/extra_field.lib.php on line 737` when the extrafields are added in the add/edit sessions